src/spandrel/architectures/OmniSR/__init__.py

```diff
@@ -9,6 +9,12 @@
 
 
 def load(state_dict: StateDict) -> SRModelDescriptor[OmniSR]:
+    # Remove junk from the state dict
+    state_dict_keys = set(state_dict.keys())
+    for key in state_dict_keys:
+        if key.endswith(("total_ops", "total_params")):
+            del state_dict[key]
```

Comment:
I think generally it's better to use strict = true so that we can catch cases where the state dict doesn't match. With strict = false, incorrect state dicts will seem like they load fine, even though they don't

Reply:
Good point. Let's keep it strict then.